custom/logistics/api.py

```diff
@@ -234,15 +247,13 @@ def sms_user_sync(self, ilsgateway_smsuser, username_part=None, password=None,
                 user.is_active = bool(ilsgateway_smsuser.is_active)
                 user.user_data = user_dict["user_data"]
                 if "phone_numbers" in user_dict:
```

Comment:
should we also check for len(user_dict["phone_numbers"]) > 0? would also be good to set phone_number = user_dict["phone_numbers"][0] once and just reference that in each place.